src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs

```diff
@@ -26,6 +26,12 @@ public class LuceneIndexingContentHandler : ContentHandlerBase
 
         private Task AddContextAsync(ContentContextBase context)
         {
+            // Check e.g if previewing.
```

Comment:
Could we check the nullability

Reply:
I'll see, normally we don't need it, a context always has a non null ContentItem, and afterwards in the deferred task it is used as is without null check.